spec/classes/puppet_profile_puppetdb_spec.rb

```diff
@@ -67,6 +67,7 @@
             :disable_ssl        => false,
             :listen_address     => '127.0.0.1',
             :ssl_listen_address => '0.0.0.0',
+            :manage_dbserver    => true,
```

Comment:
Ouch.
Maybe we need to looks at later versions of the module.
I will have a look later and se what we can do about this.